SNIClient.py

```diff
@@ -873,7 +874,8 @@ def new_check(location_id):
             if int(b) and location not in ctx.locations_checked:
                 new_check(location)
             if location in ctx.checked_locations and location not in ctx.locations_checked \
-                    and location in ctx.locations_info and ctx.locations_info[location].player != ctx.slot:
+                    and location in ctx.locations_info and ctx.locations_info[location].player != ctx.slot \
```

Comment:
in these if's the if ctx.allow_collect should come first as it's a quick bool check and might abort the contains checks early, saving some CPU. Not major, but since this is periodically running code it might make things run smoother for some people.

Reply:
Resolved in b5864bf